### PR TITLE
chore(telemetry): update endpoint and readme

### PR DESCRIPTION
**Pull request by IgnacioBecerra**
This PR updates the endpoint within each package's `telemetry.yml` file with the new stable endpoint for the collector. Also, this updates the wording used in the telemetry notice under each README.

#### Changelog

- `telemetry.yml` endpoint to new stable endpoint
- updated wording under telemetry notice in each README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,132 +38,140 @@ Then we need to include carbon-components in `src/styles.scss`:
 @use "@carbon/styles";
 ```
 
-> *Note:* For offline usage we'll need to set `$font-path: '~carbon-components/src/globals/fonts';` at the very top of our `src/styles.scss`. This will copy the fonts to our `dist` folder upon successful build. If you like the fonts to be a part of your `assets` folder and not pollute the `dist` folder then copy the fonts from `node_modules/carbon-components/src/globals/fonts` into our app's `src/assets/fonts` folder and add `$font-path: '/assets/fonts/';` at the very top of our `src/styles.scss`.
+> _Note:_ For offline usage we'll need to set `$font-path: '~carbon-components/src/globals/fonts';` at the very top of our `src/styles.scss`. This will copy the fonts to our `dist` folder upon successful build. If you like the fonts to be a part of your `assets` folder and not pollute the `dist` folder then copy the fonts from `node_modules/carbon-components/src/globals/fonts` into our app's `src/assets/fonts` folder and add `$font-path: '/assets/fonts/';` at the very top of our `src/styles.scss`.
```
Comment (Akshat55) [inline README.md:41]: @IgnacioBecerra What exactly is updated in readme? Based on what I see, there really isn't any changes apart from spacing 🤔

Comment (IgnacioBecerra) [reply README.md]: @Akshat55 Ah seems like VSCode applied some unneeded formatting when saving the file. The wording of the Telemetry notice under the README is what is different:
https://github.com/carbon-design-system/carbon-components-angular/pull/3027/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L171-L174

Comment (Akshat55) [reply README.md]: Thanks for the clarification!

**Review by Akshat55 — APPROVED** [review]
@IgnacioBecerra What's the process for viewing the telemetry data?

**Comment by IgnacioBecerra** [comment]
@Akshat55 You can reach out to `@matt` on the IBM Slack and he'll be able to get you set up for data viewing.

**Pull request merged**